### PR TITLE
Fixed the bug on the landing page

### LINKED ISSUE #100: Bug in the landing page
### Description
The upper border hides behind some div!

### Screenshots/testimonials

Screenshot from 2020-09-27 01-14-28

### PR DESCRIPTION
**Pull request by LogMoss**
Closes #100 
<Do not delete this template>
Description: I fixed the issue on the landing page where sometimes About boxes top border was hidden behind the image.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

File: client/src/components/Home/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
       height: 95vh;
```
Comment (anktrj) [inline client/src/components/Home/LandingPage/LandingPage.css]: makes this 100vh . . .  This will fix it for all screen size

**Review by ankiiitraj — CHANGES_REQUESTED** [review]

File: client/src/components/Home/LandingPage/LandingPage.css (hunk 1 above)
Comment (LogMoss) [reply client/src/components/Home/LandingPage/LandingPage.css]: Okay, but when I tested I didn't got that border issue again with any screen size

Comment (anktrj) [reply client/src/components/Home/LandingPage/LandingPage.css]: See this on iPad
Screenshot from 2020-10-01 19-53-11

Comment (LogMoss) [reply client/src/components/Home/LandingPage/LandingPage.css]: Made a commit.

**Review by EmperorYP7 — COMMENTED** [review]
The issue is not fixed here. Try to get around this some other way as suggested by @ankiiitraj.

**Comment by LogMoss** [comment]
Works after that commit
image

**Review by ankiiitraj — APPROVED** [review]

**Review by EmperorYP7 — APPROVED** [review]
Looks like this branch isn't up to date with `houseofgeeks:front-end`. Pull the changes from upstream and push to `erkska:front-end` and you're good to go!
use `git pull https://github.com/houseofgeeks/hg front-end --rebase`

**Comment by LogMoss** [comment]
Oh, I didnt saw your edit so I didnt use `--rebase`. Hope its ok

**Comment by ankiiitraj** [comment]
--rebase just helps in not creating a merge commit

**Comment by LogMoss** [comment]
so its OK?

**Comment by ankiiitraj** [comment]
@all-contributor please add @erkska for code

**Comment by EmperorYP7** [comment]
We'll be squashing those commits anyways.

**Comment by ankiiitraj** [comment]
@all-contributors please add @erkska for code

**Pull request merged**